scripts/run.sh

```diff
@@ -64,7 +64,10 @@ echo "Test pod exited with code:${podExitCode}"
 
 #Collect artifacts
 retrive-artifacts
+retrieveArtifactsExitCode=$?
+
 
 #Clean up
 delete_lease
-exit ${podExitCode}
+exitCode=$(${podExitCode}+${retrieveArtifactsExitCode})
```

Comment:
The output should tell us what command failed. Right now we fail to extract
artifacts if the test run failed so we see that output but it doesn't fail
the build because we don't capture that error codes
On Jun 13, 2016 7:07 PM, "Matthew Fisher" <notifications@github.com> wrote:

> In scripts/run.sh
> <https://github.com/deis/e2e-runner/pull/17#discussion_r66895172>:
>
> >
> >  #Clean up
> >  delete_lease
> > -exit ${podExitCode}
> > +exitCode=$(${podExitCode}+${retrieveArtifactsExitCode})
>
> Is there a way we could get both exit codes separately? If both return an
> exit code, would it be hard to tell what the exit code for each process was?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/deis/e2e-runner/pull/17/files/12f127c971138b73947da357c3b79cb33f507b69#r66895172>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAaRGBbqStGcDluPTSxRYWgrI_kqc9mbks5qLf64gaJpZM4I00B7>
> .
>